.codespellrc

```diff
@@ -0,0 +1,5 @@
+[codespell]
```

Comment:
Would you consider adding a comment in this file w/ link to docs for its syntax and format?

Reply:
Something like

```suggestion
# See https://github.com/codespell-project/codespell#using-a-config-file
[codespell]
```

?

Reply:
Perfect.  Thanks.